File: go/1159.go
package main

import "fmt"

func main() {
  var x, sum, count int

  for {
    fmt.Scanf("%d", &x)

    if x == 0 {
      break
    }

    count = 0
    sum = 0
    for {
      if x % 2 == 0 {
        count += 1
        sum += x
      }

      x += 1

      if count == 5 {
        break
      }
    }
    fmt.Println(sum)
  }
}

File: lua/1159.lua
while true do
    local x = io.read("*n")

    if x == 0 then
        break
    end

    local sum, count = 0, 0

    repeat
        if x % 2 == 0 then
            count = count + 1
            sum = sum + x
        end

        x = x + 1

        if count == 5 then
            break
        end
    until false

    print(sum)
end
